MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureVanGenuchten.h

```diff
@@ -0,0 +1,152 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on March 20, 2020, 9:59 AM
+ */
+
+#pragma once
+
+#include "MaterialLib/MPL/Property.h"
+
+namespace MaterialPropertyLib
+{
+class Medium;
+class Phase;
+class Component;
+
+/// \copydoc MaterialPropertyLib::SaturationVanGenuchten
+class CapillaryPressureVanGenuchten : public Property
+{
+public:
+    CapillaryPressureVanGenuchten(double const residual_liquid_saturation,
+                                  double const residual_gas_saturation,
+                                  double const exponent,
+                                  double const entry_pressure,
+                                  double const max_capillary_pressure);
+
+    void setScale(
+        std::variant<Medium*, Phase*, Component*> scale_pointer) override
+    {
+        if (!std::holds_alternative<Medium*>(scale_pointer))
+        {
+            OGS_FATAL(
+                "The property 'CapillaryVanGenuchten' is implemented on the "
+                "'media' scale only.");
+        }
+        this->_medium = std::get<Medium*>(scale_pointer);
+    }
+
+    /// gets \f$ p_c(S) \f$.
```

Comment:
Returns/computes/calculates sounds much better to me, and actually reflect the true functions' purpose much better.

Reply:
Changed to `It returns`.